### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by warthog9** [EvanKrall/minipcie_nvme_adapter#1]
We have some potential interest in the board, but there's no explicitly noted license, is this something that can be resolved?

**Comment by EvanKrall** [comment]
Yeah, I can add a license file. It'll be Apache 2.0 to match the license for https://github.com/mithro/kicad-mini-pci-express/blob/master/mpcie.pretty/mpcie-half-card.kicad_mod, which I modified. I can't license the 3d models, though; I should probably clean those out of the repository.

It's also worth mentioning that I haven't re-made and tested the board design since I adjusted the track widths in f754184861d658af26260aa3072ee9e9cf1eedc5 to work around https://gitlab.com/kicad/code/kicad/-/issues/11858#note_1075921362

My previous revision worked for a while and then stopped working; I haven't diagnosed why yet. I'm hoping it's just that the track width made the connection marginal, and I got lucky with initial testing, and unlucky afterwards.

**Comment by warthog9** [comment]
We shall let you know if the track width helps at all in a couple of weeks.
